RELEASE/scripts/autoscend.ash

```diff
@@ -141,7 +141,7 @@ void initializeSettings()
 	set_property("auto_forceTavern", false);
 	set_property("auto_funTracker", "");
 	set_property("auto_getBoningKnife", false);
-	set_property("auto_getStarKey", false);
+	set_property("auto_getStarKey", true);
```

Comment:
I would clean up all the places where this is set too. Ed needs it set to false but I can't think of another path where it should not default to true.

Reply:
done at commit 3c6ab4a
actually went ahead and cleaned up a few other path specific settings which were identical to the baseline settings.
this included removing the entire initialize setting function from standard, tcrs, and ocrs since those 3 were completely identical to the default settings